package.json

```diff
@@ -132,7 +132,7 @@
     "dockerode": "^2.5.5",
     "dockerode-options": "^0.2.1",
     "ejs": "^2.5.7",
-    "etcher-sdk": "^0.2.0",
+    "etcher-sdk": "^2.0.5",
```

Comment:
Nice! But also looks like a big jump, did you have a chance to test it?

Reply:
@pdcastro tests pass, will do some manual tests.

Reply:
@pdcastro just run a flash, and seems to work:

```
dependencies ~/resin/resin-cli/build-bin> sudo ./balena local flash ~/Downloads/balena-cloud-iotlondon-raspberrypi3-2.31.5+rev1-v9.11.3.img.zip
[sudo] password for greg: 
? Select a drive /dev/mmcblk0 (7.9 GB) - (resin-boot, resin-root, resin-conf, resin-data)
? This will erase the selected drive. Are you sure? Yes
Flashing [========================] 100% eta 0s  
Validating [========================] 100% eta 0s
```